index.md

```diff
@@ -5,11 +5,15 @@
 layout: home
 ---
 
-PELUX is a Linux based platform for your automotive infotainment project. The
-PELUX software architecture is optimized to secure the shortest path from
-silicon to pixel and thereby provide optimal performance to deliver a stunning
-infotainment solution. It's based on [Yocto](http://yoctoproject.org/) and
-offers an improved developer experience.
+The PELUX Base Platform is a Linux based platform used for kickstarting automotive software projects. The
```

Comment:
I think "used for kickstarting" could be misunderstood. I think it would be better to write something like "kickstart a project for developing automotive software". Just for the clarity of what we are trying to kickstart. Preferrably I would remove the "kickstart" but I can't think of a better word at the moment.